share/spice/grunt/grunt.js

```diff
@@ -0,0 +1,101 @@
+(function (env) {
+    "use strict";
+    
+    // From http://stackoverflow.com/a/34064434/1546577
+    // This is used to decode entities that is often found in the descriptions.
+    function htmlDecode(input) {
+        var doc = new DOMParser().parseFromString(input, "text/html");
```

Comment:
Oo, yeah, lemme thing of a way. Basically, anything could be in here because the description in NPM is a free country. But maybe converting a small subset is good enough.